cart/src/components/Home.js

```diff
@@ -0,0 +1,18 @@
+import CartData from "./Data";
+import "../styles/Home.css";
+
+const Home = ({ addItemToCart }) => {
+  const cards = CartData.map((items) => (
+    <div className="watch-card" key={items.id}>
+      <img src={items.logoURL} alt="" />
+      <h1>{items.heading}</h1>
+      <p>Price: {`₹${items.prices}`}</p>
```

Comment:
- I don't think both the rupee and dollar symbols are needed here.
- I also don't see the need of using string literal here, you can simply write - `<p>Price: ₹{item.price}</p>`